snuba/migrations/snuba_migrations/replays/0001_replays.py

```diff
@@ -65,7 +65,7 @@ def forwards_local(self) -> Sequence[operations.SqlOperation]:
                 columns=raw_columns,
                 engine=table_engines.ReplacingMergeTree(
                     storage_set=StorageSetKey.REPLAYS,
-                    order_by="(project_id, toStartOfDay(timestamp), cityHash64(replay_id), sequence_id)",
+                    order_by="(project_id, toStartOfDay(timestamp), cityHash64(replay_id))",
```

Comment:
This is the primary key. If we drop `sequence_id` doesn't that mean there will be many records with the same primary key? (which will get merged together since this is a replacing merge tree)

Reply:
ahh right. yes we don't want that -- so we could either timestamp in to the end of the order by, or make it a regular MergeTree. I think the only pro of a ReplacingMergeTree is we can overwrite later on if we want to remove a row, don't see any use for it otherwise.

Reply:
Changing a Table engine is not possible once you have data in them. I would recommend that you spend some time thinking which table engine you would like to use. ReplacingMergeTree seems like a decent engine which would allow you to perform replacing rows if you ever needed that functionality.

Reply:
Yeah discussed offline, many good reasons to stick with the ReplacingMergeTree. see comment above https://github.com/getsentry/snuba/pull/2972#discussion_r929357751